README.md

```diff
@@ -400,6 +401,68 @@ Example of creating regions in interleaved mode, using all NVDIMMs:
 # ipmctl create -goal PersistentMemoryType=AppDirect
 ```
 
+When running inside virtual machines, each virtual machine typically
+already gets access to one region and `ipmctl` is not needed inside
+the virtual machine. Instead, that region must be made available for
+use with `ndctl` because when the virtual machine comes up for the
+first time, the entire region is already allocated for use as a single
+block device:
+``` sh
+$ $ ndctl list -RN
+{
+  "regions":[
+    {
+      "dev":"region0",
+      "size":34357641216,
+      "available_size":0,
+      "max_available_extent":0,
+      "type":"pmem",
+      "persistence_domain":"unknown",
+      "namespaces":[
+        {
+          "dev":"namespace0.0",
+          "mode":"raw",
+          "size":34357641216,
+          "sector_size":512,
+          "blockdev":"pmem0"
+        }
+      ]
+    }
+  ]
+}
+$ ls -l /dev/pmem*
+brw-rw---- 1 root disk 259, 0 Jun  4 16:41 /dev/pmem0
+```
+
+Labels must be initialized in such a region, which must be performed
+once after the first boot:
+``` sh
+$ sudo ndctl disable-region region0
+disabled 1 region
+$ sudo ndctl init-labels nmem0
+initialized 1 nmem
+$ sudo ndctl enable-region region0
+enabled 1 region
+$ ndctl list -RN
+[
+  {
+    "dev":"region0",
+    "size":34357641216,
+    "available_size":34357641216,
+    "max_available_extent":34357641216,
+    "type":"pmem",
+    "iset_id":10248187106440278,
+    "persistence_domain":"unknown"
+  }
+]
+$ ls -l /dev/pmem*
+ls: cannot access '/dev/pmem*': No such file or directory
+```
+
+Beware that the PMEM-CSI driver will run without errors on a node
+where PMEM that was not prepared as described above. It will then
```

Comment:
That sentence sounds bit unclear, what about changing to either
1) ...on a node with PMEM that was not..
2) ...on a node where PMEM was not..

Reply:
Option 2 was what I wanted to write. Need to remove the extra "that"...